bootstrap-demo/kubernetes-dns/kubernetes/akka-cluster.yml

```diff
@@ -32,10 +32,14 @@ spec:
           httpGet:
             path: /ready
             port: 8080
+          initialDelaySeconds: 30
```

Comment:
Is it needed for readiness? That doesn't cause the node to be killed so no harm in probing right away

Reply:
I suppose it's not strictly necessary, but practically speaking if the cluster isn't stable enough where it can't say "I am alive" successfully, it doesn't seem like a good idea for it to say "I am ready for traffic."  For my own applications, I'd keep it in there.  But I can remove it if you wish for this example